pages/accounts.js

```diff
@@ -2,39 +2,127 @@ import Head from "next/head";
+import aninamtionDataMouse from "../public/lotties/mouse.json";
+import aninamtionDataStocks from "../public/lotties/stocks.json";
+import aninamtionDataLoan from "../public/lotties/loan.json";
 import Layout from "../components/Layout";
+import styled from "styled-components";
 
 export default function Accounts() {
   const {user} = useContext(UserContext);
 
+  const accountName = user?.accounts?.map(account => account.name);
+
   // default Options for Lottie animation
-  const defaultOptions = {
+  const defaultOptionsUnderConstruction = {
+    loop: true,
+    autoplay: true,
+    animationData: animationDataUnderConstruction,
+    rendererSettings: {
+      preserveAspectRatio: "xMidYMid slice",
+    },
+  };
+
+  const defaultOptionsPiggyAccount = {
+    loop: true,
+    autoplay: true,
+    animationData: aninamtionDataPiggyAccount,
+    rendererSettings: {
+      preserveAspectRatio: "xMidYMid slice",
+    },
+  };
+
+  const defaultOptionsMouse = {
+    loop: true,
+    autoplay: true,
+    animationData: aninamtionDataMouse,
+    rendererSettings: {
+      preserveAspectRatio: "xMidYMid slice",
+    },
+  };
+
+  const defaultOptionsStocks = {
     loop: true,
     autoplay: true,
-    animationData: animationData,
+    animationData: aninamtionDataStocks,
     rendererSettings: {
       preserveAspectRatio: "xMidYMid slice",
     },
   };
+
+  const defaultOptionsLoan = {
+    loop: true,
+    autoplay: true,
+    animationData: aninamtionDataLoan,
+    rendererSettings: {
+      preserveAspectRatio: "xMidYMid slice",
+    },
+  };
+
   return (
     <>
       <Head>
         <title>kidsFi - Finance for kids</title>
       </Head>
       <Layout />
       {user?.isChild ? (
-        <p style={{padding: "5rem"}}>
-          Hold your horses, I&apos;m working on it
-        </p>
+        user.accounts?.length !== 0 ? (
+          <StyledAnimationContainer>
+            {accountName?.includes("Piggy bank") && (
+              <Lottie
+                options={defaultOptionsPiggyAccount}
+                width={"5rem"}
+                height={"5rem"}
+              />
+            )}
+
+            {accountName?.includes("Savings account") && (
+              <Lottie
+                options={defaultOptionsMouse}
+                width={"5rem"}
+                height={"5rem"}
+              />
+            )}
+
+            {accountName?.includes("Stocks account") && (
+              <Lottie
+                options={defaultOptionsStocks}
+                width={"5rem"}
+                height={"5rem"}
+              />
+            )}
+
+            {accountName?.includes("Loan account") && (
+              <Lottie
+                options={defaultOptionsLoan}
+                width={"5rem"}
+                height={"5rem"}
+              />
+            )}
+          </StyledAnimationContainer>
+        ) : (
+          <p>
+            You don&apos;t have any accounts yet. Please ask your parents to
```

Comment:
not sure, if the content of this <p> Tag is meant to be this way. If so ignore this comment.